guides/V1_space_template_main.md

```diff
@@ -0,0 +1,31 @@
+---
+copyright: 'Copyright IBM Corp. 2018'
+link: 'delete-space-template'
+is: 'experimental'
+---
+---
+
+# Space Template
+
+In Watson Work Services a space template is a template applied to a team space on creation of the space.
+The space template defines user configurable properties for the space, 
+as well as a set of required apps that are automatically added to the space on creation.
+
+# Properties
```

Comment:
How about presenting the structure of the template (and other relevant types) in tabular format as, for example, in [V1_spaces_main](https://github.com/watsonwork/watsonwork-developer-docs/blob/master/guides/V1_spaces_main.md)?

Reply:
Might be a good idea. I would prefer to do this just for the main ones described here instead of documenting fields that should be clear from the schema itself, as it is more work to maintain and more likely the documentation here will become outdated.